src/main/java/org/fundacionjala/core/config/EnvironmentProperties.java

```diff
@@ -0,0 +1,41 @@
+package org.fundacionjala.core.config;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public final class EnvironmentProperties {
+
+    /**
+     * Constructor of the Class.
+     */
+    private EnvironmentProperties() {
+
+    }
+    /**
+     * Initialize a new GetProperties.
+     * @return GetProperties.
+     */
+    public static EnvironmentProperties getInstance() {
```

Comment:
Were you considering to change this class to be singleton? Because it lools like that, but you are always returning a new instance

Reply:
Are you using this method?